MapboxDirections/MBRouteStep.swift

```diff
@@ -527,7 +529,7 @@ public class RouteStep: NSObject, NSSecureCoding {
         destinations = json["destinations"] as? String
         
         let maneuver = json["maneuver"] as! JSONDictionary
-        instructions = maneuver["instruction"] as! String
+        instructions = maneuver["instruction"] as? String ?? "\(maneuverType) \(maneuverDirection)"
```

Comment:
`maneuverType` and `maneuverDirection` are optional, so string interpolation will turn them into e.g. `Optional(turn) Optional(left)`. Instead, use an `if let` statement to conditionally populate `instructions` using unwrapped values.